_posts/2018-07-20-exploring-state-management-in-react-container-components.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: post
+title: Exploring State Management in React&#58; Container Components
+date: 2018-07-20 09:00:00 -04:00
+categories: post
+desc: An amazing blog post, truly one of the best! PS CHANGEME
+permalink: /:categories/:year/:month/:day/:title/
+---
+
+>When should we install Redux?
```

Comment:
The question threw me briefly. I think what you're really asking is something like "at what point should the complexity of our solution drive us toward redux?" or something similar?